roles/firewall/templates/compute_iptables.j2

```diff
@@ -0,0 +1,21 @@
+*filter
+:INPUT ACCEPT [2:104]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [2:204]
+-A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+-A INPUT -p icmp -j ACCEPT
+-A INPUT -i lo -j ACCEPT
+-A INPUT -p tcp -m multiport --dports 22 -m comment --comment "allow ssh incoming" -j ACCEPT
+{% for node in groups[controllers] %}
```

Comment:
In each 'for' clause you will need single quotes around the group name.  See, https://docs.ansible.com/playbooks_variables.html#magic-variables-and-how-to-access-information-about-other-hosts